src/components/Inputs/input.module.scss

```diff
@@ -2,6 +2,17 @@
     position: relative;
     width: fit-content;
 
+    input[type='submit'],
```

Comment:
this needs to be in search box as well ??

Reply:
What's the scope?